fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/ChunkWriter.java

```diff
@@ -110,6 +110,8 @@ private void pushFhirJsons2Cos(InputStream in, int dataLength) throws Exception
             throw new Exception("pushFhirJsons2Cos: no cosClient!");
```

Comment:
looking at this Exception (Perhaps there should be a FHIRBulkException... not sure in this specific case, as this sits in another framework... 

As is said... Food for thought...

Reply:
I think this is not needed.